invenio_indexer/version.py

```diff
@@ -12,4 +12,4 @@
 and parsed by ``setup.py``.
 """
 
-__version__ = '1.2.7'
+__version__ = '1.2.8'
```

Comment:
missing CHANGES.rst.

is this backwards compatible? if you had indeces created with documents with `_doc`, having now `None`. Would it work?

Reply:
I would prefer to nump minor honestly, it is safer, ping @slint 
I think otherwise we might break cds-videos (potentially, I don't remember which ES version we have there)

Reply:
Not against it as long as is backwards compatible (I'm not 100% sure, maybe worth a check).

Otherwise, we risk breaking any other instance out (e.g. CERN search)  there and need `.post` releases to fix (i.e. break semver)

Reply:
It's breaking ES6 for sure. But otherwise it shouldn't cause issues on ES7 (doc types are not allowed anyways, so passing `_doc` or `None` shouldn't matter, irrelevant of if you created documents with one or another), and AFAIK we don't have any instances running on ES6 at the moment.

I agree that bumping minor would be the safest though, as in "we're 99.9% sure this is backwards compatible (for ES7), but we can't test everything".